OpenDreamRuntime/Procs/DMOpcodeHandlers.cs

```diff
@@ -430,9 +430,19 @@ private static void HandleSuffixPronoun(ref StringBuilder formattedString, ReadO
 
         public static ProcStatus? ListAppend(DMProcState state) {
             DreamValue value = state.Pop();
-            DreamList list = state.Pop().GetValueAsDreamList();
+            DreamValue list = state.Pop();
+            switch (list.Type) {
+                case DreamValue.DreamValueType.Float: {
+                    break;
+                }
+                case DreamValue.DreamValueType.String:
+                    break;
+                case DreamValue.DreamValueType.DreamObject: {
+                    IDreamMetaObject metaObject = list.GetValueAsDreamObject().ObjectDefinition.MetaObject;
+                    metaObject?.OperatorAppend(list, value, state);
+                    break;
+                }
```

Comment:
you forgor a extra `}`

Reply:
you're jumping the gun a lot here, there's about a dozen methods and I need to refactor and associated compile errors yet before I can even test it!

Reply:
smh